pages/services/beta-storage/0.3.2-beta/tutorials/index.md

```diff
@@ -0,0 +1,362 @@
+{
+    "name": "devices-provider",
+    "description": "Expose devices on a node",
+    "spec": {
+        "plugin": "devices",
+        "node": "ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0",
+        "plugin-configuration": {
+            "blacklist": "loop[0-9]"
+        }
+    }
+}
+EOF
+$ dcos storage provider create devices-provider.json
+```
+
+If the creation is successful, you can view the newly created volume provider using the `list` command.
+
+```bash
+$ dcos storage provider list
+PLUGIN   NAME              NODE                                     STATE
+devices  devices-provider  ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0  ONLINE
+```
+
+3. List the devices on the node.
+**Note:** All devices that are not blacklisted will be listed, regardless of whether they are already in use or not.
+
+```bash
+$ dcos storage device list --node=ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0
+NODE                                     NAME   STATUS  ROTATIONAL  TYPE   FSTYPE  MOUNTPOINT
+ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0  usr    ONLINE  false       crypt  ext4    /usr
+ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0  xvda   ONLINE  false       disk   -       -
+ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0  xvda1  ONLINE  false       part   vfat    /boot
+ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0  xvda2  ONLINE  false       part   -       -
+ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0  xvda3  ONLINE  false       part   ext4    -
+ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0  xvda4  ONLINE  false       part   -       -
+ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0  xvda6  ONLINE  false       part   ext4    /usr/share/oem
+ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0  xvda7  ONLINE  false       part   -       -
+ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0  xvda9  ONLINE  false       part   ext4    /
+```
+
+4. Let's assume that device `xvda4` and device `xvda7` are unused.
+Create an LVM volume provider using both devices.
+
+```bash
+$ cat > volume-group-1.json <<EOF
+{
+    "name": "volume-group-1",
+    "description": "The primary volume group",
+    "spec": {
+        "plugin": "lvm",
+        "node": "ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0",
+        "plugin-configuration": {
+            "devices": ["xvda4", "xvda7"]
+        },
+        "labels": {"rotational": "false"}
+    }
+}
+EOF
+$ dcos storage provider create volume-group-1.json
+```
+
+If the creation is successful, you can view the newly created volume provider using the `list` command.
+
+```bash
+$ dcos storage provider list
+PLUGIN   NAME              NODE                                     STATE
+devices  devices-provider  ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0  ONLINE
+lvm      volume-group-1    ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0  ONLINE
+```
+
+The logs for all LVM providers on a node may be viewed using the [DC/OS CLI](../latest/cli/command-reference/dcos-node/dcos-node-log/):
+```bash
+$ dcos node log --mesos-id=ca626e4a-e3cb-4613-b7dd-618cfc19bee1-S0 --filter=CSI_PLUGIN:csilvm --filter=STREAM:STDERR
```

Comment:
This `dcos node log` example isn't new for 0.3.2 but it **is** hard to find. I'd like to find a better, more discoverable home for this kind of stuff at some point in the future.

Reply:
Open to suggestions...